### PR TITLE
[SMALLFIX] Clean up Java warnings

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

**Comment by aaudiber** [comment]
Jenkins, test this please.

**Comment by gpang** [comment]
Jenkins, test this please

File: shell/src/main/java/alluxio/shell/AlluxioShell.java [hunk 1]
```diff
@@ -43,15 +43,13 @@
...
-  private static final HashMap<String, String[]> CMD_ALIAS = new HashMap<String, String[]>() {
-    {
-      put("chgrpr", new String[] {"chgrp", "-R"});
-      put("chmodr", new String[] {"chmod", "-R"});
-      put("chownr", new String[] {"chown", "-R"});
-      put("lsr", new String[] {"ls", "-R"});
-      put("rmr", new String[] {"rm", "-R"});
-    }
-  };
+  private static final Map<String, String[]> CMD_ALIAS = ImmutableMap.<String, String[]> builder()
```
Comment (gpang) [inline shell/src/main/java/alluxio/shell/AlluxioShell.java]: checkstyle complains about: error: GenericWhitespace '>' is followed by whitespace.

Comment (aaudiber) [inline shell/src/main/java/alluxio/shell/AlluxioShell.java]: fixed, thanks

**Comment by aaudiber** [comment]
Jenkins, test this please.

**Comment by yupeng9** [comment]
LGTM. Thanks for the improvement.

**Pull request merged**